src/manager/InMemoryHistoryManager.java

```diff
@@ -1,25 +1,88 @@
 package manager;
 
 import task.Task;
-import java.util.ArrayList;
-import java.util.List;
 
-public class InMemoryHistoryManager implements HistoryManager{
-    private final List<Task> tasksHistory;
-    static final int MAX_VALUE = 10;
+import java.util.*;
+
+public class InMemoryHistoryManager implements HistoryManager {
+
+    public static class Node {
+        public Task item;
+        public Node next;
+        public Node prev;
+
+        public Node(Node prev, Task element, Node next) {
+            this.prev = prev;
+            this.item = element;
+            this.next = next;
+        }
+    }
+
+    private final HashMap<Integer, Node> history;
```

Comment:
Обозначать нужно интерфейс, чтобы сделать код более гибким и не зависящим от конктретной реализации коллекции
```suggestion
    private final Map<Integer, Node> history;
```